### PR TITLE
Update plugins, add maven site

### PR DESCRIPTION
**Pull request by hwbllmnn**
@terrestris/devs Please review.

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!--
+  ~ SHOGun GPRO, based on https://terrestris.github.io/shogun/
+  ~
```
Comment (marcjansen) [inline src/site/site.xml:3]: This is not correct

File: src/site/site.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ~     https://www.apache.org/licenses/LICENSE-2.0.txt
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+<project name="GPRO Webgis">
```
Comment (marcjansen) [inline src/site/site.xml:19]: Same here

**Review by marcjansen — CHANGES_REQUESTED** [review]

File: src/site/site.xml (hunk 1 above)
Comment (hwbllmnn) [reply src/site/site.xml:3]: Right, fixed in 159eeb7

File: src/site/site.xml (hunk 2 above)
Comment (hwbllmnn) [reply src/site/site.xml]: Right, fixed in 159eeb7

**Review by KaiVolland — APPROVED** [review]

**Pull request merged**